cmd/goal/README.md

```diff
@@ -86,83 +102,27 @@ goal account balance --address ${APP_ACCOUNT}
 
 ### Q: How do I use boxes in goal? In particular, I'd like to make a goal app call which:
 * accesses a particular box for a particular app
-* the name of the box is defined as an ABI type
-* displays the results as JSON
+* stores an ABI type as its contents
 
 ### A:
 Here's an example with the following assumptions:
 
-* the callers's account is given by `$ACCOUNT` (see first answer)
-* the program used was created in as in the second question. In particular:
-  * it routes to box subroutines using the method signifier `app-arg 0`
-* the app id has been stored in `$APPID` (see the previous answer)
-* the box referenced in the first app-call has name `greatBox`
-* another referenced box has abi-name of type `(byte,byte,byte)` and name `[2,3,5]`
-* this second box is provided contents `42`
-* `msgpacktool` is used to decode to JSON
-* it is assumed that `$ALGORAND_DATA` has been exported (see previous answer)
+* the callers's account is given by `${ACCOUNT}` (see first answer)
+* the program used is `boxes.teal` referenced above. In particular:
+  * it routes to box subroutines using the app argument at index 0 as the method signifier
+* the app id has been stored in `${APPID}` (see the previous answer)
+* the box referenced in the first non-create app-call has name `greatBox`
+* another referenced box is named `an_ABI_box`
+  * this second box is provided contents `[2,3,5]` of ABI-type `(uint8,uint8,uint8)`
+  * we provide the box'es `${APPID}` as well in the `--box` reference. This can be used to access boxes of external applications
```

Comment:
@jannotti - do you agree with this claim?

Reply:
So I'll remove the wrong explnation and the comma notation as it's not needed for this simple example

Reply:
Yes, I think they can learn that later. This is just going to get them up and running a simple box using app.